litellm/tests/test_alangfuse.py

```diff
@@ -228,6 +228,40 @@ async def test_langfuse_logging_without_request_response(stream, langfuse_client
         pytest.fail(f"An exception occurred - {e}")
 
 
+@pytest.mark.asyncio
+async def test_langfuse_masked_input_output(langfuse_client):
+    """
+    Test that creates a trace with masked input and output
+    """
+    import uuid
+
+    for mask_value in [True, False]:
+        _unique_trace_name = f"litellm-test-{str(uuid.uuid4())}"
+        litellm.set_verbose = True
+        litellm.success_callback = ["langfuse"]
+        response = await create_async_task(
```

Comment:
shouldn't this be `await litellm.acompletion` ?

Reply:
If you look above in the test file, this is a helper method that will essentially create an async task for litellm.acompletion, but it just has some defaults. I can use litellm.acompletion directly if you would prefer that.

Reply:
nope just wanted to clarify